docs/pages/workflows/control-flow.mdx

```diff
@@ -67,3 +67,7 @@ jobs:
       profile: production
       platform: ios
 ```
+
+## Combining conditions
+
+You can use both `needs` and `if` conditions in the same job definition. Note that the job will be run if all jobs on the `needs` list are successful **AND** the `if` condition is true. If any job on the `needs` list had been skipped or the `if` condition is false, the current job will be skipped. If any job on the `needs` list had failed, the current job will be failed.
```

Comment:
```suggestion
You can use both `needs` and `if` conditions in the same job definition. **Note that the job will be run if all jobs on the `needs` list are successful and the `if` condition is true.** If any job on the `needs` list has been skipped or the `if` condition is false, the current job will be skipped. If any job on the `needs` list has failed, the current job will be failed.
```

- Let's highlight the sentence that talks about the note since it is important enough, instead of highlighting the word
- Fix grammatical typo